src/main/java/com/programmers/smrtstore/domain/order/presentation/dto/res/OrderResponse.java

```diff
@@ -1,5 +1,60 @@
 package com.programmers.smrtstore.domain.order.presentation.dto.res;
 
+import com.programmers.smrtstore.domain.order.domain.entity.Order;
+import com.programmers.smrtstore.domain.order.domain.entity.enums.OrderStatus;
+import com.programmers.smrtstore.domain.order.domain.entity.vo.PaymentInfo;
+import com.programmers.smrtstore.domain.user.presentation.dto.res.ProfileUserResponse;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
 public class OrderResponse {
 
+    private String id;
+
+    private OrderStatus orderStatus;
+
+    private Integer totalPrice;
+
+    private String orderDate;
+
+    private PaymentInfo paymentInfo;
+
+    private String deliveryInfo;
+
+    private List<OrderedProductResponse> products;
+
+    private ProfileUserResponse orderer;
+
+    @Builder
```

Comment:
정적 팩토리 메서드 내부에서만 사용되는 Builder라면 외부에서 사용하지 못하도록 접근자를 주는 것은 어떨까요?

```suggestion
    @Builder(access = AccessLevel.PRIVATE)
```

Reply:
> 정적 팩토리 메서드 내부에서만 사용되는 Builder라면 외부에서 사용하지 못하도록 접근자를 주는 것은 어떨까요?
> 
> 
> 
> ```suggestion
> 
>     @Builder(access = AccessLevel.PRIVATE)
> 
> ```


dto 라서 외부에서 test 시 생성할 일이 있을것같아 열어두었습니다!

Reply:
👍🏻